cluster/partitioner/partitioner.go

```diff
@@ -4,49 +4,64 @@ import (
 	"fmt"
 
 	"github.com/Shopify/sarama"
+	"github.com/cespare/xxhash"
+	jump "github.com/dgryski/go-jump"
 	"github.com/raintank/schema"
 )
 
 type Partitioner interface {
-	Partition(schema.PartitionedMetric, int32) (int32, error)
+	Partition(schema.PartitionedMetric, int32) int32
```

Comment:
Is this interface still used anywhere? same question in the tsdb code btw. can't see where it's used.

Reply:
i was wondering the same thing and looked around a bit, and also didn't see it being used anywhere. i'll just remove it.